### PR TITLE
Addresses NGDOC-150

### PR DESCRIPTION
**Pull request by marcfishelquantil**
I added Authentication Certificate with a description and renamed Verify Origin to Verify Origin Certificate.

### CODE REVIEW

File: en/docs/portal/edge-configurations/managing-origins.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Origins are your web servers containing the content you want CDN Pro to accelera
...
 | Supported Protocol   | Protocol(s) that your origin server supports. Choices are: <ul><li><strong>HTTP</strong> = use HTTP only.<li><strong>HTTPS</strong> = use HTTPS only.<li><strong>Both</strong> = use HTTP and HTTPS. (*default*)</ul>|
-| Verify Origin         | Specify whether CDN Pro should verify the TLS certificate on the origin servers. Choices are:<ul><li><strong>True</strong> = perform backend verification.<li><strong>False</strong> = do not perform backend verification. (*default*)</ul>|
+| Verify Origin Certificate        | Specify whether CDN Pro should verify the TLS certificate on the origin servers. Choices are:<ul><li><strong>True</strong> = perform backend verification.<li><strong>False</strong> = do not perform backend verification. (*default*)</ul>|
+| Authentication Certificate        |Select a certificate you want to use to authenticate with the origin server. The certificate must be created and deployed like other certificates. Because this certificate is used to connect to your origin server, you would typically upload your own certificate instead of creating a CDN Pro auto-renewed certificate.</ul>|
```
Comment (burtonquantil) [inline en/docs/portal/edge-configurations/managing-origins.md:30]: @marcfishelquantil  Let's get rid of the referenceto  the auto-renewed certificate and just point the user to https://docs.cdnetworks.com/en/cdn/docs/portal/certificates/creating-certificates#uploading-an-existing-certificate

The certificate must be uploaded and then deployed like other certificates.

uploaded can link to https://docs.cdnetworks.com/en/cdn/docs/portal/certificates/creating-certificates#uploading-an-existing-certificate


"deployed" should link to https://docs.cdnetworks.com/en/cdn/docs/portal/certificates/deploying-certificates

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Let's simplify the text for verify origin certificate
Is there supposed to be a screenshot of the origin configuration dialog in https://docs.cdnetworks.com/en/cdn/docs/portal/edge-configurations/managing-origins
?

File: en/docs/portal/edge-configurations/managing-origins.md (hunk 1 above)
Comment (marcfishelquantil) [reply en/docs/portal/edge-configurations/managing-origins.md:30]: Good points. fixes made to the English- and Chinese-language files.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Comment by marcfishelquantil** [comment]
Merging approved edits.

**Pull request merged**